docs-app/types/glint-registry.d.ts

```diff
@@ -1,8 +1,8 @@
 import '@glint/environment-ember-loose';
 import '@glint/environment-ember-template-imports';
-// Types from libraries
-import '@crowdstrike/ember-oss-docs/glint';
```

Comment:
side-effecting imports are no longer recommended for merging the glint registry, because the number of copies of `@glint/environment-ember-loose/registry` in a project's `node_modules` is too variable.